marc_to_solr/lib/traject_config.rb

```diff
@@ -1335,7 +1335,9 @@
   values = []
   alma_852(record).each do |field|
     subfields = field.subfields.reject { |s| s.value.empty? }.collect { |s| s if ["h", "i"].include?(s.code) }.compact
+    next unless subfields.present?
```

Comment:
Could we use `next if subfields.empty?` instead?

Reply:
yes I can change this.